CHANGELOG.md

```diff
@@ -2929,6 +2929,21 @@ Other minor changes
                     f Preserves₂ ≈₁ ⟶ ≈₁ ⟶ ≈₂
   ```
 
+* Added new proofs to `Relation.Binary.Construct.Closure.Transitive`:
+  ```
+  accessible⁻ : ∀ {x} → Acc _∼⁺_ x → Acc _∼_ x
+  wellFounded⁻ : WellFounded _∼⁺_ → WellFounded _∼_
+  ```
+  as instances of the corresponding proofs in `Induction.WellFounded.Subrelation`,
+  together with a refactoring of the former proof
```

Comment:
I don't think the refactoring of the proof needs to be mentioned, nor the implementation of the new proofs. Otherwise the CHANGELOG will become very long indeed!